### DISCUSSION TITLE
valheim crossplay.

### DISCUSSION THREAD

**Discussion by hizpain** [GameServerManagers/LinuxGSM#4066] [Q&A]
Some can help me to able crossplay in my server.

**Comment by irobot73** [comment]
Using latest LGSM:
- Stop server via console
- Update your .CFG to enable CROSSPLAY
- Via console, install missing dependencies (originally noted [HERE](https://www.reddit.com/r/valheim/comments/xshlyk/dedicated_server_on_linux_crossplay_switch_not/))
`sudo apt update && sudo apt install libatomic1 libpulse0 libpulse-dev`
- Start server via console

Should see entries in the LOG noting successful PlayFab+ integration.  Long as you don't rebuild the container, you should be good through any reboots (or redo the steps)

```
09/06/2024 06:12:41: Server 'Double-D's Valheim Server' begin PlayFab create and join network for server 
09/06/2024 06:12:43: Joined PlayFab Party network with ID "...
...
09/06/2024 06:12:43: Session "Double-D's Valheim Server" registered with join code ...
```

Enjoy
